### PR TITLE
Probable condescending language removed!

### PR DESCRIPTION
**Pull request by mananchawla2005**

### CODE REVIEW

File: docs/04_Concepts.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Fulfillment    |
 #### Client side
 
 The public key is to be used with requests that don't require any sensitive actions or data to be retrieved. The
-Commerce.js SDK includes all client side endpoints, so you can use them quickly and easily in your client-facing
+Commerce.js SDK includes all client side endpoints, so you can use them in your client-facing
```
Comment (robbieaverill) [inline docs/04_Concepts.md:54]:
```suggestion
Commerce.js SDK includes all client side endpoints, so you can use them quickly and easily in your client-facing
```

I don't agree that this is condescending

File: docs/04_Concepts.md [hunk 2]
```diff
@@ -77,8 +77,7 @@ sensitive request would be to make a call to [refund an order](/docs/api/?shell#
 </div>
 
 When you register for a Chec account, you'll be assigned two sets of these keys: live and sandbox. We highly recommend
-using your sandbox key until you're ready to deploy your new project live. Orders created with sandbox keys can easily
-be cleared from the Chec Dashboard, and will automatically use the "Test Gateway" for payment processing.
+using your sandbox key until you're ready to deploy your new project live. Orders created with sandbox keys can be cleared from the Chec Dashboard, and will automatically use the "Test Gateway" for payment processing.
```
Comment (robbieaverill) [inline docs/04_Concepts.md:80]:
```suggestion
using your sandbox key until you're ready to deploy your new project live. Orders created with sandbox keys can easily
be cleared from the Chec Dashboard, and will automatically use the "Test Gateway" for payment processing.
```

There's no improvement here

File: docs/05_Full-SDK-Reference.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ position: 5
 category: Full SDK reference
 ---
 
-**Commerce.js** is a JavaScript SDK built on top of the **Chec Platform** to allow for easy interfacing with the **Chec
+**Commerce.js** is a JavaScript SDK built on top of the **Chec Platform** to allow for interfacing with the **Chec
```
Comment (robbieaverill) [inline docs/05_Full-SDK-Reference.md:8]:
```suggestion
**Commerce.js** is a JavaScript SDK built on top of the **Chec Platform** to allow for easy interfacing with the **Chec
```

**Review by robbieaverill — CHANGES_REQUESTED** [review]
Thanks for the PR. I'm happy to remove the "just" words in the getting started page, but the rest of your edits seem superfluous.

**Review by robbieaverill — APPROVED** [review]
Thanks

**Comment by mananchawla2005** [comment]
Always happy to help 🙂. A small request also. Kindly add hacktoberfest accepted label to this pr as it was not related to any issue. It was independent so as per rules label is required.

**Comment by robbieaverill** [comment]
Jump over and get some Chec swag while you're at it!

https://commercejs.com/docs/hacktoberfest

**Comment by mananchawla2005** [comment]
@robbieaverill yeah but dont have debit/credit card. Anyways loved contributing :-)

**Comment by robbieaverill** [comment]
@mananchawla2005 it's free, you don't need a card!

**Pull request merged**